sdks/nodejs/package-lock.json

```diff
@@ -2514,6 +2514,51 @@
             "engines": {
                 "node": ">=10"
             }
+=======
```

Comment:
This file looks like it has merge conflicts in it. Please make sure you are using nodejs 16 when running the command (or alternately just edit the version line by hand).

See https://github.com/googleforgames/agones/pull/2465

Reply:
oh shoot, I thought this was fixed last night. Thanks for picking that up, I missed it late at night.